cloud/fallback-public-cloud.yaml

```diff
@@ -54,31 +56,41 @@ clouds:
     description: Google Cloud Platform
     auth-types: [ jsonfile, oauth2 ]
     regions:
-      us-east1:
+      asia-east1:
```

Comment:
prefer the original ordering with the us regions at the top etc

Reply:
yeah I thought that might be the case.. will fix up